### ISSUE TITLE
Keyboard navigation for UI (mainly for forms)

### ISSUE THREAD

**Issue by webmaxru** [GoogleDeveloperExperts/experts-app-web#93]
It's important to provide mouse free UX for submitting and editing activities. `tabindex`-based navigation is the main part of it. For now after pressing `TAB` the focus doesn't go to the next field but disappears. Has to be fixed for all forms (main) and other UI elements.

**Comment by justinribeiro** [comment]
Just as an FYI, this is a known issue in `iron-overlay-behavior` (there is a PR sitting that hasn't been reviewed that resolves this: https://github.com/PolymerElements/iron-overlay-behavior/pull/200). There is not a good workaround currently (focus is hard), but once that PR lands we should be golden.

**Comment by LostInBrittany** [comment]
Yeah, as Justin said, I'm kinda been waiting for that fix for a moment...

The only workaroround I could do now it some hand coded focus routine for the form, and since [ PolymerElements/iron-overlay-behavior#200](https://github.com/PolymerElements/iron-overlay-behavior/pull/200) should arrive to master soon, I'd prefer to wait for it...

**Comment by justinribeiro** [comment]
I say we wait. :-) My two cents.

**Comment by LostInBrittany** [comment]
I'm not sure it will be ready for 2.0.0-Beta, so I'm removing the label...

**Comment by justinribeiro** [comment]
Worst case, we update the dep to point to that hash and ship the PR (without it being in mainline).

**Comment by justinribeiro** [comment]
I've pushed a PR that will need further testing and probably a few tweaks, but it basically gets us an okay keyboard experience (make sure to run `bower update` and select the hash version for `iron-overlay-behavior`). Click to see the video below on YouTube showing what it looks like in action.

[![IMAGE ALT TEXT](https://img.youtube.com/vi/QGWkpoSc5CM/0.jpg)](https://youtu.be/QGWkpoSc5CM)

It is not the end-all-be-all, but it works around the problem will discovering others (particularly that `paper-button` with `disabled` pegged to a property basically makes tabindex not work which required me to some validation on the fields, or that `paper-menu-dropdown` can't take `focus()` which is really heart warming). I'm not going to go into the full list of somewhat random things I had to do to make this smooth but the code will tell the tale.

Comments (and testing before merge) very very welcome. :-)
